CMake/kwiver-utils.cmake

```diff
@@ -18,4 +18,5 @@ include("${CMAKE_CURRENT_LIST_DIR}/utils/kwiver-utils-modules.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/utils/kwiver-utils-sphinx.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/utils/kwiver-utils-tests.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/utils/kwiver-utils-python.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/utils/kwiver-utils-msvc.cmake")
```

Comment:
Should this be conditional on MSVC build?

Reply:
Do you like the msvc check in this file or in the kwiver-utils-msvc.cmake file?
(I kind of like it in the file, so anybody using it, its already there)
but I am not strong on this position...

Reply:
I'm inclined to put the check before the include, in this file.